docs/pages/repo/docs/core-concepts/monorepos/configuring-workspaces.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Configuring Workspaces
+description: Turborepo lets you configure each workspace
+---
+
+import Callout from "../../../../../components/Callout";
+
+# Configuring Workspaces
+
+Most monorepos can declare a `turbo.json` in the root directory with a uniform
```

Comment:
This felt a little...maybe presumptuous? Not sure what word I want to use.

At any rate, I think we can jump straight to "While uniformity across your codebase tends to make things easier to handle, not all packages and workspaces are built the same. To accommodate this..."

Reply:
Talked about this internally and sticking with this, but open to word smithing!